src/low-preview/ci.styles.css

```diff
@@ -50,3 +50,65 @@ img.ci-image-ratio.ci-image-preview {
 .ci-bg.ci-bg-animation:before {
   filter: blur(10px);
 }
+
+/* Zoom & close icons */
+.ci-close-icon,
+.ci-zoom-icon {
+  padding: 10px;
+  position: absolute;
+  cursor: pointer;
+}
+
+.ci-close-icon {
+  top: 0;
+  right: 0;
+}
+
+/* Zoom icon */
+.ci-zoom-icon {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 900;
+  background-color: #e3e3e3;
+  opacity: 0;
+  transition: opacity 200ms ease-in;
+}
+
+.ci-image-wrapper:hover .ci-zoom-icon {
+  opacity: 0.8;
+}
+
+/* Modal */
+.ci-modal {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 999;
+  background-color: #434242dc;
+}
+
+.ci-modal-image-wrapper {
+  width: 80%;
+}
+
+.ci-modal-image-wrapper .ci-modal-image,
+.ci-modal-image-wrapper .ci-image-wrapper {
+  height: 80vh !important;

```

Comment:
why it's important if it's overridden by the demo styles you need to check the demo stylesheet and fix it there because actually by that Implementation the user can't override the styles you added